### PR TITLE
Update readme w/ less packages required to avoid confusion

### PR DESCRIPTION
**Pull request by wooj2**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,42 +56,26 @@ You should use one of these three ways to import the AWS Mobile SDK but not mult
         $ sudo gem install cocoapods
         $ pod setup
 
-2. In your project directory (the directory where your `*.xcodeproj` file is), create a plain text file named `Podfile` (without any file extension) and add the lines below. Replace `YourTarget` with your actual target name.
+2. In your project directory (the directory where your `*.xcodeproj` file is), run the following to create a `Podfile` in your.
```
Comment (kneekey23) [inline README.md:59]: In your -> in your project

File: README.md [hunk 2]
```diff
@@ -54,42 +56,26 @@ You should use one of these three ways to import the AWS Mobile SDK but not mult
...
-        
-    ![image](readme-images/cocoapods-setup-01.png?raw=true)
+        $ pod init
+
+3. Edit the podfile to include the pods you want to integrate into your project.  For example, if you need auth, you can use AWSMobileClient, and if you need analytics, you add AWSPinpoint.  As a result, your podfile might look something like this:
+
+target 'YourTarget' do
+    pod 'AWSMobileClient'
+    pod 'AWSPinpoint'
+end
```
Comment (lawmicha) [inline README.md:68]: need to wrap in code block

File: README.md [hunk 3]
```diff
@@ -106,28 +92,16 @@ You should use one of these three ways to import the AWS Mobile SDK but not mult
...
+4. With your project open in Xcode, select your **Target**. Under **General** tab, find **Frameworks, Libraries, and Embedded Content** and then click the **+** button.
 
-5. Click the **Add Other...** button, navigate to the `AWS<#ServiceName#>.framework` files under `Carthage` > `Build` > `iOS` and select them. Do not check the **Destination: Copy items if needed** checkbox when prompted.
+5. Click the **Add Other...** button, then "Add Files..." in the popup menu, then navigate to the `AWS<#ServiceName#>.framework` files under `Carthage` > `Build` > `iOS` and select them. Do not check the **Destination: Copy items if needed** checkbox if prompted.  Add the frameworks that you need for you specific use case.  For example, if you are using AWSMobileClient and AWSPinpoint, you will want to add the following frameworks:
 
-    * `AWSCore.framework`
-    * `AWSAutoScaling.framework`
-    * `AWSCloudWatch.framework`
-    * `AWSCognito.framework`
+    * `AWSAuthCore.framework`
```
Comment (lawmicha) [inline README.md:99]: how do i know i need this just from needing AWSMobileClient?

Comment (wooj2) [reply README.md:100]: Spoke offline, resolving.

**Pull request merged**